framework/messages/el/yii.php

```diff
@@ -1,4 +1,4 @@
-<?php
+// <?php
```

Comment:
?

Reply:
Sorry, I don't know how this // got it. Fixed it.